teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(47, 'teacher')
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    @age > 18 || @parent_permission == true
```

Comment:
- [ ] This method should always return `true` kindly make sure to implement this